IDsAboveHead/cl_idsabovehead.lua

```diff
@@ -39,10 +49,16 @@ Citizen.CreateThread(function()
                     if playerDistances[id] < disPlayerNames then
                         local targetPedCords = GetEntityCoords(targetPed)
                         if NetworkIsPlayerTalking(id) then
-                            DrawText3D(targetPedCords, GetPlayerServerId(id), 247,124,24)
-                            DrawMarker(27, targetPedCords.x, targetPedCords.y, targetPedCords.z-0.97, 0, 0, 0, 0, 0, 0, 1.001, 1.0001, 0.5001, 173, 216, 230, 100, 0, 0, 0, 0)
+                            if displayid then
+                                DrawText3D(targetPedCords, GetPlayerServerId(id), 52, 152, 219)
+                            end
+                            if Config.Blip then
+                                DrawMarker(27, targetPedCords.x, targetPedCords.y, targetPedCords.z-0.97, 0, 0, 0, 0, 0, 0, 1.001, 1.0001, 0.5001, 173, 216, 230, 100, 0, 0, 0, 0)
+                            end
                         else
-                            DrawText3D(targetPedCords, GetPlayerServerId(id), 255,255,255)
+                            if AlwaysDisplayID then

```

Comment:
The `else` here is meant to show the text in white if they are **not talking**, now it looks like it will only show if they are talking

Reply:
That's the point. For my server, I have it that way. It only appears when they speak. So I decided to make it configurable here so if others who cannot exactly code but share the idea can have it.